cfpb/consumerfinance.gov
# Filters: Add filter count logic to change column width based on number of filters present

## Changes

- Add `filter_count` variable to track number of filters in the first row of multiselect filters.
- Change column width based on `filter_count` value.
- Add some code comments in the jinja template.


## How to test this PR

1. Visit the following filters pages and compare to localhost. They should have wider first row multiselect filters when there are less than 3 filters:

https://www.consumerfinance.gov/activity-log/

https://www.consumerfinance.gov/about-us/newsroom/

https://www.consumerfinance.gov/about-us/blog/

https://www.consumerfinance.gov/enforcement/actions/

https://www.consumerfinance.gov/administrative-adjudication-proceedings/administrative-adjudication-docket/

https://www.consumerfinance.gov/compliance/supervisory-guidance/

https://www.consumerfinance.gov/data-research/research-reports/

https://www.consumerfinance.gov/rules-policy/final-rules/

https://www.consumerfinance.gov/enforcement/petitions/

https://www.consumerfinance.gov/foia-requests/foia-electronic-reading-room/

https://www.consumerfinance.gov/consumer-tools/everyone-has-a-story/

https://www.consumerfinance.gov/rules-policy/rules-under-development/

https://www.consumerfinance.gov/data-research/cfpb-researchers/

https://www.consumerfinance.gov/consumer-tools/credit-reports-and-scores/consumer-reporting-companies/companies-list/

https://www.consumerfinance.gov/about-us/events/

https://www.consumerfinance.gov/rules-policy/notice-opportunities-comment/open-notices/

https://www.consumerfinance.gov/rules-policy/notice-opportunities-comment/archive-closed/

https://www.consumerfinance.gov/compliance/amicus/briefs/

https://www.consumerfinance.gov/about-us/events/archive-past-events/


## Screenshots

Before:
<img width="904" alt="Screen Shot 2021-06-29 at 11 17 46 AM" src="https://user-images.githubusercontent.com/704760/123824069-b7e30000-d8cb-11eb-82d5-09248a52f130.png">

After:
<img width="905" alt="Screen Shot 2021-06-29 at 11 17 58 AM" src="https://user-images.githubusercontent.com/704760/123824071-b87b9680-d8cb-11eb-9e01-ac694d99ee5f.png">
